src/accelerate/utils/environment.py

```diff
@@ -48,3 +50,12 @@ def parse_flag_from_env(key, default=False):
 def parse_choice_from_env(key, default="no"):
     value = os.environ.get(key, str(default))
     return value
+
+
+def are_libraries_initialized(library_names: list = []) -> Dict[str, bool]:
+    """
+    Checks if any of `library_names` are imported in the environment. Will return results as a `key:bool` pair.
+    """
+    if isinstance(library_names, str):
+        library_names = [library_names]
+    return {lib_name: lib_name in sys.modules for lib_name in library_names}
```

Comment:
The code here and at the call site could be simplified if only the list of loaded libraries were returned, instead of returning all libs in a dict with their status as value. But maybe you plan on using this for other purposes later which requires all the results, so not sure.